### ISSUE TITLE
Move into raven-go?

### ISSUE THREAD

**Issue by mattrobenolt** [evalphobia/logrus_sentry#5]
Hey @evalphobia thanks for doing this!

Would you be opposed to moving this into raven-go? We have some other framework support, like Gin, that we're likely going to be moving into raven-go as well.

That way you'd do something like:

```go
import "github.com/getsentry/raven-go/logrus"
```

Then we'd get documentation going over at https://docs.getsentry.com/hosted/clients/go/integrations/

Thoughts?

**Comment by mattrobenolt** [comment]
Not 100% sure how we'd want to do this, but ideally this `logrus_sentry.NewSentryHook(YOUR_DSN, ...)` installation too since the default client inside `raven-go` should already exist and we'd leverage that instead.

I'd envision something like:

```go
raven.SetDSN("...")
log := logrus.New()
hook := raven_logrus.NewHook([]logrus.Level{
    logrus.PanicLevel,
    logrus.FatalLevel,
    logrus.ErrorLevel,
})
log.Hooks.Add(hook)
```

**Comment by evalphobia** [comment]
@mattrobenolt 
Sounds good! 
If you officially supports sentry library for logrus, it's more stable and be awared by gophers.

@gigaroby @awonak @miloconway 
any thoughts?

@miloconway 
If you're re-creating the [PR in Sirupsen/logrus](https://github.com/Sirupsen/logrus/pull/192), I'll wait for it. 
or if you need more time, I gonna move this repository before your PR.

/cc @Sirupsen FYI

**Comment by miloconway** [comment]
@evalphobia I'll recreate the diff.

**Comment by gigaroby** [comment]
@evalphobia no problem :)

**Comment by evalphobia** [comment]
@miloconway @gigaroby 
Thanks guys.

@mattrobenolt
When you ready, tell me that?
After that, we have to do...

+ Mark this repository as a deprecated one.
+ Modify the link on [README of sentry Hook](https://github.com/Sirupsen/logrus/blame/4643a7efec9b8fd7548882e7b30095711343100f/README.md#L208) in [Sirupsen/logrus](https://github.com/Sirupsen/logrus)
+ :beers:

**Comment by stkao05** [comment]
@evalphobia @mattrobenolt 
Would love seeing this happen. If needed, I could help working on that on my fork of `raven-go` and create PR for you to merge it.

**Comment by mzuneska** [comment]
Just curious if there was there any progress on this? I don't see any mention of logrus in the `raven-go` repository.

**Comment by mattrobenolt** [comment]
There has not. I'm still open to inclusion here. :) Just need someone to submit a PR.

**Comment by mzuneska** [comment]
@mattrobenolt I opened a basic PR in the raven-go repo. It's referenced above.

**Comment by mattrobenolt** [comment]
Awesome, I will get to look at that this week. A bit busy at the moment. :) Thanks! <3

**Comment by captncraig** [comment]
I would be opposed to this btw. I am using this to send to sentry, but not using any other sentry-specific code in my app. I appreciate that this package is small and lightweight, and prefer it being separate.

**Comment by mattrobenolt** [comment]
@captncraig how would this change affect that? Just curious. This package directly depends on `raven-go` already. There's no way you could use this without that dependency. Merging these repositories would just mean they're under the same roof instead of two dependencies.

**Comment by captncraig** [comment]
You are right. I feel silly having said that now, since of course this already depends on raven-go. Sorry I chimed in.

**Comment by ellisonleao** [comment]
hey guys, any updates on this?

**Comment by macnibblet** [comment]
and we enter into 2018

**Comment by evalphobia** [comment]
WIP https://github.com/getsentry/raven-go/pull/98

**Comment by marcosox** [comment]
still WIP I suppose
